packages/viewer/src/components/viewer-dom-element/viewer-dom-element.tsx

```diff
@@ -198,6 +198,39 @@ export class ViewerDomElement implements HTMLDomRendererPositionableElement {
   @Event({ bubbles: true })
   public occlusionStateChanged!: EventEmitter<boolean>;
 
+  /**
+   * Disables detached testing for this element. Defaults to enabled. When
+   * enabled, the elements position will be tested against the current depth
+   * buffer. If the position is detached, then the `detached` attribute will be
+   * set.
+   */
+  @Prop()
+  public detachedOff = false;
```

Comment:
You'd want what sounds like an experimental feature to be opt-in, no?

Reply:
Would also recommend we communicate somewhere in the docs that this is experimental.

Reply:
Good idea, updated!